lib/fluent_migrations/firefox/all-unified.py

```diff
@@ -0,0 +1,163 @@
+                value=REPLACE(
+                    all_unified,
+                    "Test your sites against soon-to-be-released Firefox browser features with powerful, flexible DevTools that are on by default.",
+                    {
+                        "Firefox": TERM_REFERENCE("brand-name-firefox"),
+                    }
+                )
+            ),
+            FTL.Message(
+                id=FTL.Identifier("firefox-all-count-on-stability-and"),
+                value=REPLACE(
+                    all_unified,
+                    "Count on stability and ease of use with this Firefox browser built for enterprise.",
+                    {
+                        "Firefox": TERM_REFERENCE("brand-name-firefox"),
+                    }
+                )
+            ),
+            FTL.Message(
+                id=FTL.Identifier("firefox-all-windows-installers-for"),
+                value=REPLACE(
+                    all_unified,
+                    "Windows installers for corporate IT that simplify the configuration, deployment and management of the Firefox Browser.",
+                    {
+                        "Firefox Browser": TERM_REFERENCE("brand-name-firefox-browser"),
+                    }
+                )
+            ),
+            FTL.Message(
+                id=FTL.Identifier("firefox-all-choose-a-32-bit-installer"),
+                value=REPLACE(
+                    all_unified,
+                    "Choose a 32-bit installer for computers with 32-bit processors — or for older or less powerful computers. <a href=\"%(url)s\">If you aren’t sure</a> whether to choose a 64-bit or 32-bit installer, we recommend you go with 32-bit.",
+                    {
+                        "%%": "%",
+                        "%(url)s": VARIABLE_REFERENCE("url"),
+                    }
+                )
+            ),
+            FTL.Message(
+                id=FTL.Identifier("firefox-all-download-product-for"),
+                value=REPLACE(
+                    all_unified,
+                    "Download %(product_label)s for %(platform)s in %(locale)s",
+                    {
+                        "%%": "%",
+                        "%(product_label)s": VARIABLE_REFERENCE("product_label"),
+                        "%(platform)s": VARIABLE_REFERENCE("platform"),
+                        "%(locale)s": VARIABLE_REFERENCE("locale"),
+                    }
+                )
+            ),
+        ] + transforms_from("""
+firefox-all-msi-installers = {COPY(all_unified, "MSI installers",)}
+firefox-all-which-version = {COPY(all_unified, "Which version would you like?",)}
+firefox-all-select-your-preferred-installer = {COPY(all_unified, "Select your preferred installer",)}
+firefox-all-select-your-preferred-language = {COPY(all_unified, "Select your preferred language",)}
+firefox-all-learn-about-installers = {COPY(all_unified, "Learn about installers",)}
```

Comment:
You could add the plain ftl strings from https://github.com/mozilla/bedrock/pull/8785/files#diff-dc49c5b09feb9ddd6e22466a59ba6484R56-R63 here at the end of the recipe to add them to all translated files. Re https://github.com/mozilla-l10n/www-l10n/pull/27#issuecomment-616451159